### ISSUE TITLE
Hazard Nuker [HELP]

### ISSUE THREAD

**Issue by WojtekPlay** [Rdimo/Hazard-Nuker#69]
Is it safe? cause my antivirus is saying its a virus "Win64:MalwareGen" the exe file

**Comment by Rdimo** [comment]
yes it's safe, hazard nuker is open source so you can see what it does, + if you want you can check the exe by decompiling it if you know how to do that since I didnt obfuscate it or anything, I just have the exe version for people so they dont have to install all the dependencies and its easier for me to just tell them to run the exe instead of telling them step by step on how to install all dependencies and download python etc..

**Comment by WojtekPlay** [comment]
ok thxs

**Comment by Rdimo** [comment]
np mate

**Comment by Awire9966** [comment]
If you want, you can use [7zip](http://7zip.org/) to open the exe and see whats in it.

**Comment by NightTerror5293** [comment]
Hi, I am facing a problem in the code,
![image](https://user-images.githubusercontent.com/88367912/149059711-39d8fab7-9d53-4fd6-a853-55d240abd84c.png)
can u pls help me solve it?

**Comment by Rdimo** [comment]
xd that aint hazard, but it looks like you need to import the winreg dependency and then just import all the modules from it

**Comment by NightTerror5293** [comment]
uh,the hazard nuker also has some problem too 
![image](https://user-images.githubusercontent.com/88367912/149153680-34e96e38-3047-4f7d-a597-42f46421a436.png)

**Comment by Awire9966** [comment]
First off why do you guys have lightmode XD ?

**Comment by NightTerror5293** [comment]
ans my question 1st pls
am in danger

**Comment by NightTerror5293** [comment]
hello
?

**Comment by Rdimo** [comment]
xd @Awire9966 some people do be like that

**Comment by Rdimo** [comment]
and to fix your issue @NightTerror5293 just open cmd adn type `pip install --force-reintsall undetected_chromedriver`

**Comment by Ixogamer** [comment]
`--force-reinstall` *

**Comment by Rdimo** [comment]
shi spelt it wrong ty

**Comment by NightTerror5293** [comment]
k
ThANK U!

**Comment by NightTerror5293** [comment]
uh,can anyone help how to use the token grabber?

**Comment by Ixogamer** [comment]
I honestly recommend you use [Rdimos' other token grabber](https://github.com/Rdimo/Hazard-Token-Grabber-V2). It's pretty well documented.
